### ISSUE TITLE
Buzzard Bait - BOOT ERROR with .woz image

### ISSUE THREAD

**Issue by tomcw** [AppleWin/AppleWin#938]
As reported by @GreatHierophant via Slack.

https://archive.org/details/wozaday_Buzzard_Bait

Reproduced with Enhanced //e, //e and II+.

**Comment by tomcw** [comment]
Looks like the same issue as #921

Triage/development note:

1. Comment out this: (and with significantBitCells = 50): boots OK
```
	if (i == bitCellRemainder-1)			// On last bitCell
		m_latchDelay += extraLatchDelay;	// +0 or +2
```

NB. the code in 1, is the alt fix to Wasteland here: https://github.com/AppleWin/AppleWin/issues/733#issuecomment-576026713

**Comment by tomcw** [comment]
Fixed by #930 / b5726aa.
